lib/index.js

```diff
@@ -102,6 +105,15 @@ module.exports = function (gulp, options) {
     runWithDependencyProcess(runMocha, _.flattenDeep([options.unitTestFiles, options.integrationTestFiles]), cb);
   });
 
+  gulp.task('test-junit', 'Run all the tests with code coverage and output the results as JUNIT. (Specify ', ['lint-junit'], function (cb) {
+    var file = process.env.MOCHA_FILE || './build/test/test-results.xml';
+    var dir = path.dirname(file);
+    if (!fs.existsSync(dir)) fs.mkdirSync(dir);
+    options.reporter = 'mocha-junit-reporter';
```

Comment:
I'm not so sure on mutating the options like this instead of creating a new object with these properties. Although it won't cause a problem in the majority of cases, if we ever have a task that combines multiple of these tasks, having a task modify the options in the middle of things could cause surprising behavior.